src/components/listItems/assignmentListItem.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { Assignment } from 'devu-shared-modules'
+
+import styles from './assignmentListItem.scss'
+import { prettyPrintDate } from 'utils/date.utils'
+
+import ListItemWrapper from 'components/shared/layouts/listItemWrapper'
+
+type Props = {
+  assignment: Assignment,
+  courseId: string
+}
+
+const AssignmentListItem = ({ assignment, courseId }: Props) => (
+  <ListItemWrapper
+    to={`/courses/${courseId}/assignments/${assignment.id}`}
+    tag={`${courseId}-${assignment.id}`}>
+      <h3 className={styles.name}>{assignment.name}</h3>
```

Comment:
Under normal circumstances I don't think you'd want to use a header in a list item, even if the sizing is correct. It make screen readers go wonky when you're misusing "headers"